src/azure-account.ts

```diff
@@ -460,7 +463,7 @@ export class AzureLoginHelper {
 			environment,
 			userId: tokenResponse.userId!,
 			tenantId: tokenResponse.tenantId!,
-			credentials: new DeviceTokenCredentials({ environment: environment, username: tokenResponse.userId, clientId, tokenCache: this.delayedCache, domain: tokenResponse.tenantId })
+			credentials: new DeviceTokenCredentials(clientId, tokenResponse.tenantId, tokenResponse.userId, undefined, defaultEnvironment, this.tokenCache)
```

Comment:
Why pass `defaultEnvironment` and not the `environment`?

Reply:
Nice catch.